extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -345,6 +347,44 @@ export class AmpA4A extends AMP.BaseElement {
 
     /** @private {string} */
     this.safeframeVersion_ = DEFAULT_SAFEFRAME_VERSION;
+
+    /** @private @const {!./refresh-manager.RefreshManager}  */
+    this.refreshManager_ = refreshManagerFor(this.win);
```

Comment:
Negative here is that networks that do not require refresh will include its code in their binary unnecessarily.  We should consider how to avoid this long term but given that only Doubleclick wants to support this currently, perhaps we should consider how to make it directly a feature in its impl.